python/ray/util/spark/utils.py

```diff
@@ -14,6 +15,12 @@ def is_in_databricks_runtime():
     return "DATABRICKS_RUNTIME_VERSION" in os.environ
 
 
+def _convert_dbfs_path_to_local_path(dbfs_path):
+    parsed_path = urlparse(dbfs_path)
+    assert parsed_path.scheme.lower() == "dbfs", "dbfs path is required."
+    return "/dbfs" + parsed_path.path
```

Comment:
```suggestion
    return "file:///dbfs" + parsed_path.path
```

Reply:
Is this needed?

Reply:
Is the DBFS act like NFS so all nodes in the cluster can read it as local files? Please make sure it's working as expected in multi-nodes cluster, thanks.

Reply:
> Is the DBFS act like NFS so all nodes in the cluster can read it as local files?

Yes.